spec/workers/reactions/update_reactable_worker_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe Reactions::UpdateReactableWorker, type: :worker do
+  describe "#perform" do
+    let(:article) { create(:article) }
+    let(:reaction) { create(:reaction, reactable: article) }
+    let(:comment) { create(:comment, commentable: article) }
+    let(:comment_reaction) { create(:reaction, reactable: comment) }
+    let(:worker) { subject }
+
+    it " updates the reactable Article" do
+      expect do
+        worker.perform(reaction.id)
+      end. to have_enqueued_job(Articles::ScoreCalcJob).exactly(:once).with(article.id)
+    end
+
+    it " updates the reactable Comment" do
```

Comment:
Can you remove these leading spaces from the spec descriptions? Just want to keep things tidy :)